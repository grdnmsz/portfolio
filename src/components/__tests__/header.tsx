import React from 'react'
import renderer from 'react-test-renderer'

import { Header } from '../'

describe('Header', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
