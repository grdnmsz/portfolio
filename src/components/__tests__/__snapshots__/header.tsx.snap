// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders correctly 1`] = `
<nav
  className="bg-headerColor"
>
  <div
    className="max-w-7xl mx-auto px-2 sm:px-6 md:px-20"
  >
    <div
      className="relative flex items-center justify-between h-16"
    >
      <div
        className="absolute inset-y-0 left-0 flex items-center sm:hidden"
      >
        <button
          aria-expanded="false"
          className="inline-flex items-center justify-center p-2 rounded text-gray-300 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
          onClick={[Function]}
        >
          <span
            className="sr-only"
          >
            Open main menu
          </span>
          <svg
            aria-hidden="true"
            className="block h-6 w-6"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4 6h16M4 12h16M4 18h16"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
            />
          </svg>
          <svg
            aria-hidden="true"
            className="hidden h-6 w-6"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M6 18L18 6M6 6l12 12"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
            />
          </svg>
        </button>
      </div>
      <div
        className="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start"
      >
        <div
          className="flex-shrink-0 flex items-center "
        >
          <a
            href="/"
          >
            <div
              className="text-xl sm:mr-12"
            >
              gordon
              <span
                className="text-teal-700"
              >
                $&gt;
              </span>
              dev
            </div>
          </a>
        </div>
        <div
          className="hidden sm:block sm:ml-6"
        >
          <div
            className="flex space-x-12"
          >
            <a
              className="py-2 block text-sm font-medium"
              href="/"
            >
              Home
            </a>
            <a
              className="py-2 text-sm font-medium"
              href="/about"
            >
              About
            </a>
            <a
              className="py-2 text-sm font-medium"
              href="/articles"
            >
              Articles
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    aria-checked={false}
    className="hidden sm:hidden"
  >
    <div
      className="px-2 pt-2 pb-3 space-y-1"
    >
      <a
        className="px-3 py-2 block rounded text-sm font-medium"
        href="/"
      >
        Home
      </a>
      <a
        className="px-3 py-2 block rounded text-sm font-medium"
        href="/about"
      >
        About
      </a>
      <a
        className="px-3 py-2 block rounded text-sm font-medium"
        href="/articles"
      >
        Articles
      </a>
    </div>
  </div>
</nav>
`;
